deeprank/learn/DataSet.py

```diff
@@ -25,8 +25,7 @@
 class DataSet():
 
     def __init__(self, train_database, valid_database=None, test_database=None,
-                 chain1='A', chain2='B',
-                 mapfly=True, grid_info=None,
+                 grid_info=None,
                  use_rotation=None,
                  select_feature='all', select_target='DOCKQ',
```

Comment:
We probably won't have users deciding what target they want, as the program will predict.

Reply:
It's used in the methods "get_norm" and "load_one_variant" though. Wouldn't it be better to set it to something else?
Like: "class".

Reply:
Yes, but if we set select_target='class' but we must make sure this option is not modifiable for the user. So yes, I'll make sure readme file clarifies this. 
normalize_target must always be False.

Also, in line 892 and 893, since we have 0.0 and 1.0 when program tries to normalize- it will try dividing 0 by 1 so this might be problematic when both min and max values are 0.

Reply:
I removed the normalize target feature in this commit: https://github.com/DeepRank/deeprank/commit/fb5f304b9a0c9f508db78f9233ea27083fb8538c

Reply:
great